sscanss/app/dialogs/insert.py

```diff
@@ -1204,6 +1206,25 @@ def updateCrossSection(self):
         self.showBounds(self.bounds_button.isChecked())
         self.clearShape()
 
+    def highlightPoints(self, rows):
+        """Highlights the points corresponding to the rows currently selected in the point manager
+        
+        :param rows: the currently highlighted rows
+        :type rows: List[bool]
+        """
+        items = self.scene.items()
+        fixed_points = {item.rank: item for item in items if isinstance(item, GraphicsPointItem) and item.fixed}
+
+        for i, row in enumerate(rows):
+            if fixed_points.get(i):
+                if row:
+                    fixed_points[i].default_pen = self.scene.path_pen
```

Comment:
I missed this previously changing the default pen is not necessary
only `fixed_points[i].highlighted = row` is needed and maybe `row` is not the best name

Reply:
ok, so perhaps `row_selected` is preferable to `row`?